pom.xml

```
@@ -223,5 +223,30 @@ limitations under the License.
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>plexus-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>enforce-java</id>
+                <goals>
+                  <goal>enforce</goal>
+                </goals>
+                <configuration>
+                  <rules>
+                      <requireJavaVersion>
+                          <version>11</version>
+                      </requireJavaVersion>
+                  </rules>
```

With the risk of being a PITA, I think the identing is a bit off compared to the rest of the file
```suggestion
                  <rules>
                    <requireJavaVersion>
                      <version>11</version>
                    </requireJavaVersion>
                  </rules>
```

---

Good catch! My IDE tricked me... Thanks for proposing this change!